taverna-cwl-utilities/CWLFiles/customtool1.cwl

```diff
@@ -0,0 +1,67 @@
+################################################################################
```

Comment:
Could we move `CWLFiles/` to `src/test/resources` to avoid dependency on correct relative paths?  (E.g. so it works in both the IDE and Maven).

Then within the test you can try to use `getClass().getResource()` to retrieve it.. or perhaps that is awkward for methods that need a `Path`?